pysph/base/octree.pyx

```diff
@@ -621,23 +689,23 @@ cdef class Octree:
 
         cdef int num_threads = get_number_of_threads()
 
-        #Use the parallel method
-        if ( (num_threads > 4 and num_particles > 40000) or test_parallel ):
+        # Use the parallel method
+        if ( (num_threads > 1) or test_parallel ):
             self.method = 1
```

Comment:
Is this self.method hack needed anymore given that there is just a single test_parallel case.